### DISCUSSION TITLE
Using the starting blog as a template

### DISCUSSION THREAD

**Discussion by MrGKanev** [wanoo21/tailwind-astro-starting-blog#11] [Q&A]
Hi,

This may be an already talked about issue somewhere, but I wanted to ask how can we use the starter blog and make changes keeping the possibility of easy updates.

**Comment by wanoo21** [comment]
Hi @MrGKanev, fork this repo, and run it as your blog, when there are new changes, you'll be able to merge them instantly from this repo 🙂

**Comment by MrGKanev** [comment]
Hi @wanoo21 but what about changes that I want to make to the template? Example - adding packages and etc. It may have too many merge conflicts.

**Reply by wanoo21** [reply]
I'm not sure if there's a better way than this, at least you'll see the differences and will be able to resolve the conflicts using git, which I think is the best

**Reply by MrGKanev** [reply]
In other frameworks (for example Laravel) you can over write over files so you can keep the updates easy and make changes to your files when you want. I still can't find a way in Astro.
